cases/src/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -161,38 +145,22 @@ protected synchronized void getCases() {
         }
         objectIdMapping = new Hashtable<>();
         cases = new Vector<>();
-        if (caseRecords != null) {
-            int i = 0;
-            for (String id : caseRecords) {
-                cases.addElement(new CaseChildElement(this, -1, id, i));
-                ++i;
-            }
-        } else {
-            int mult = 0;
-            for (IStorageIterator i = storage.iterate(); i.hasMore(); ) {
-                int id = i.nextID();
-                cases.addElement(new CaseChildElement(this, id, null, mult));
-                objectIdMapping.put(DataUtil.integer(id), DataUtil.integer(mult));
-                mult++;
-            }
-
+        int mult = 0;
+        for (IStorageIterator i = storage.iterate(); i.hasMore(); ) {
+            int id = i.nextID();
+            cases.addElement(new CaseChildElement(this, id, null, mult));
+            objectIdMapping.put(DataUtil.integer(id), DataUtil.integer(mult));
+            mult++;
         }
     }
 
-    public void setState(String syncToken, String stateHash) {
-        this.syncToken = syncToken;
-        this.stateHash = stateHash;
-    }
```

Comment:
Appears to be unused. Removed all code associated with `syncToken` and `stateHash` because these are the only places they are set.